website/content/blog/2023-11-reduce-reuse-rebase-buildpacks.md

```diff
@@ -0,0 +1,47 @@
+---
+title: "Reduce, Reuse, Rebase: Sustainable Containers with Buildpacks"
+description: Buildpacks are the greenest choice for container builds
+slug: 2023-11-reduce-reuse-rebase-buildpacks
+weight: 1
+date:   2023-11-17 00:00:00 +0000
+author: Joe Kutner
+---
+
+Container builds can be very wasteful. Every operating system update, new dependency version, and toolchain upgrade results in copious amounts of energy used to build and rebuild our container images; often unnecessarily. It can be costly at scale, which is why Cloud Native Buildpacks were designed to perform builds as efficiently as possible.
+
+Buildpacks are the greenest choice for container builds because the rebaseable layers and advanced caching mechanisms they provide can prevent many images from being rebuilt at all. That's a big shift from many cloud native technologies that assume unlimited resources are available in the cloud.
```

Comment:
Again, please remove such claims as "the greenest choice."

Reply:
Certainly. What about this?

> Buildpacks are an environmentally friendly choice for container builds ...